Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:alpine AS build
```

Comment:
Should we pin to a Python version?

Reply:
Might be better. Maybe 3.8 will break something. I don't think so, but best practice.